### ISSUE TITLE
Frontend product sorting via POST?

### ISSUE THREAD

**Issue by tzyganu** [magento/magento2#656]
This is more a question than an issue, but it might be an issue.  
I asked the same thing on [magento.stackexchange](http://magento.stackexchange.com/q/35208/146).  
Is there a reason to why the frontend product list sorting is done via POST instead of GET?  
In my opinion is more user friendly to have it via GET. This way the customer can bookmark a link or share it with the products already sorted his desired criteria.

**Comment by vpelipenko** [comment]
Internal ticket: MAGETWO-28212

**Comment by choukalos** [comment]
Hi @tzyganu - this is on our radar for near term development ( within a month ).  We got this feedback from our tech forum and have a few stories to address them ( + MAGETWO-31677 ).  You'll notice that we weren't terribly consistent with our REST route naming in addition to the PUTs vs GETs for search.

Originally we went with PUT as we thought the search criteria query was too complex and could run over the 2k character limit that IE8 has.  After feedback we decided to change them to GETs and put an epic in for post 2.0 GA to revise how we handle the search criteria parameter as it's a tad wordy to create a query with a mix of and's and or's.  It would be (hint hint) awesome if someone wanted to submit a PR to make the search criteria parameter a bit more user friendly.

Closing since we've got this tracked internally and ready to be done by Dev RC.
